pkg/apis/core/validation/validation.go

```diff
@@ -5601,6 +5604,44 @@ func validatePodResourceClaimStatuses(statuses []core.PodResourceClaimStatus, po
 	return allErrs
 }
 
+// validatePodExtendedResourceClaimStatus validates the ExtendedResourceClaimStatus in a pod status.
+func validatePodExtendedResourceClaimStatus(status *core.PodExtendedResourceClaimStatus, containers []core.Container, fldPath *field.Path) field.ErrorList {
+	if status == nil {
+		return nil
+	}
+	var allErrs field.ErrorList
+	rmFldPath := fldPath.Child("requestMappings")
+	containerExtended := make(map[string]struct{})
```

Comment:
prefer structured keys to avoid string allocs and for clarity:
```go
	type key struct {
		container string
		resource  string
	}
	seen := map[key]struct{}{}
```

Reply:
done